interface/patient_file/report/patient_report.php

```diff
@@ -86,456 +86,464 @@ function show_date_fun(){
 ?>
```

Comment:
Add some space after the navbar so it doesn't look like it's colliding
![image](https://user-images.githubusercontent.com/46497068/91070401-ac1c1280-e604-11ea-976c-c96eb0edb583.png)